Library/Formula/tiff2png.rb

```diff
@@ -2,25 +2,15 @@
 
 class Tiff2png < Formula
   homepage 'http://www.libpng.org/pub/png/apps/tiff2png.html'
-  url 'ftp://ftp.simplesystems.org/pub/libpng/png/applications/tiff2png/tiff2png-0.91.tar.gz'
-  sha1 '3a23abaaadbed8f3d13b88241257fe2078eb61fd'
-  revision 1
+  url 'https://people.xiph.org/~giles/tiff2png/tiff2png-0.92.tar.gz'
+  sha256 'bf743ae5b231d690202eb8a552f173619815533172417aeb3f8a75e13315f0ed'
```

Comment:
I don't think making this adversarial helps anyone. If you mean your "This file being secure relies on Git's SHA1s being secure," I was asking you to elaborate because it makes no sense to me. If I can't understand the reasoning behind it, I'm not going to be a efficient contributor.

What's your threat model, where compromising github is no harder than compromising the source download from a third-party site? I'd think the later would be much easier. There are more options for one thing. Some of those will be less well protected.